packages/manager/modules/pci/src/projects/project/serving/namespace/models/models.html

```diff
@@ -0,0 +1,76 @@
+<cui-message-container data-messages="$ctrl.messages"></cui-message-container>
+<div>
+    <p class="mb-3">
+        <span data-translate="pci_projects_project_serving_namespace_models_description_1"
+              data-translate-values="{
+                  description: $ctrl.project.description
+              }">
+        </span>
+        <span data-translate="pci_projects_project_serving_namespace_models_description_2"></span>
+    </p>
+    <oui-datagrid data-rows="$ctrl.models" data-row-loader="$row">
+        <oui-column data-title=":: 'pci_projects_project_serving_namespace_models_id' | translate"
+                    data-property="id"
+                    data-type="string"
+                    data-searchable
+                    data-filterable></oui-column>
+
+        <oui-column data-title=":: 'pci_projects_project_serving_namespace_models_version_status' | translate"
+                    data-property="versionStatus">
+            <span class="oui-status"
+                    data-ng-class="{
+                    'oui-status_success' : $row.versionStatus === $ctrl.VERSION_STATUS.deployed,
+                    'oui-status_warning' : $row.versionProcessing(),
+                    'oui-status_error' : $row.versionError()
+                    }"
+                    data-translate="{{:: 'pci_projects_project_serving_namespace_models_version_status_' + $row.versionStatus }}">
+            </span>
+        </oui-column>
+        <oui-column data-title=":: 'pci_projects_project_serving_namespace_models_api_status' | translate"
+                    data-property="apiStatus">
+            <span class="oui-status"
+                  data-ng-class="{
+                    'oui-status_success' : $row.apiStatus === $ctrl.API_STATUS.running,
+                    'oui-status_warning' : $row.apiStatus !== $ctrl.API_STATUS.running
+                    }"
+                  data-translate="{{:: 'pci_projects_project_serving_namespace_models_api_status_' + $row.apiStatus }}">
+            </span>
+        </oui-column>
+        <oui-column data-title=":: 'pci_projects_project_serving_namespace_models_url' | translate"
+                    data-property="url"
+                    data-type="string"
+                    data-searchable
+                    data-filterable>
+            <a href="{{$row.url}}" target="_blank">{{$row.url}}</a>
```

Comment:
```suggestion
            <a data-ng-href="{{$row.url}}" target="_blank">{{$row.url}}</a>
```
A good practice for `target="_blank"` links is to also add a `rel=noopener`

Reply:
9eb0962dad0527f7e808bd6b11ebe3c3cfec136e